scripts/omDownloadInputs.py

```diff
@@ -22,14 +22,17 @@
 This downloads the input files that rarely change and can be cached between runs.
 """
 
-from omInputs import (electricityPath,
+from openmethane_prior.omInputs import (electricityPath,
                       # fugitivesPath,
                       landUsePath, sectoralEmissionsPath, sectoralMappingsPath, ntlPath, auShapefilePath, livestockDataPath, termitePath, wetlandPath, coalPath, oilGasPath)
 import requests
 import os
-from omUtils import getenv
+from openmethane_prior.omUtils import getenv
 
 root_path = os.path.dirname(os.path.realpath(__file__))
+# go one up because this file now lives in the scripts folde
+# TODO! We could also use Pathlib for this step.
+root_path = os.path.abspath(os.path.join(root_path, os.pardir))
```

Comment:
```suggestion
scripts_path = os.path.dirname(os.path.realpath(__file__))
# go one up because this file now lives in the scripts folder
# TODO! We could also use Pathlib for this step.
root_path = os.path.abspath(os.path.join(scripts_path, os.pardir))
```
Nice. I wasn't aware about `os.pardir` and have always sprinkled `..`'s everywhere.
I agree that this should be moved to pathlib which is easier to understand